atlasdb-workload-server-distribution/src/main/java/com/palantir/atlasdb/workload/WorkloadServerLauncher.java

```diff
@@ -111,11 +120,30 @@ private void runWorkflows(WorkloadServerConfiguration configuration, Environment
         }
     }
 
+    private WorkflowValidator<Workflow> createRingWorkflowValidator(
+            AtlasDbTransactionStoreFactory transactionStoreFactory,
+            RingWorkflowConfiguration workflowConfig,
+            LifecycleEnvironment lifecycle) {
+        ExecutorService executorService =
+                lifecycle.executorService(RingWorkflows.class.getSimpleName()).build();
+
+        InteractiveTransactionStore transactionStore = transactionStoreFactory.create(
+                Map.of(
+                        workflowConfig.tableConfiguration().tableName(),
+                        workflowConfig.tableConfiguration().isolationLevel()),
+                Set.of());
+        return WorkflowValidator.of(RingWorkflows.create(
+                transactionStore,
+                workflowConfig,
+                MoreExecutors.listeningDecorator(executorService),
+                new AtomicBoolean(false)));
```

Comment:
I missed this in the previous review - but would it make sense to change `RingWorkflows.create` to not take a boolean at all?

It seems like we want skip running to always be false at the start